xcp_d/utils/plot.py

```diff
@@ -1061,3 +1062,25 @@ def _get_tr(img):
     except AttributeError:  # Error out if not in cifti
         return img.header.get_zooms()[-1]
     raise RuntimeError("Could not extract TR - unknown data structure type")
+
+
+def plot_reho_mosaic_volumetric(reho, template):
```

Comment:
I don't think you need ReHo- and ALFF-specific functions for this. A generic `plot_stat_map` call should be fine.